python/singa/image_tool.py

```diff
@@ -264,6 +295,45 @@ def resize_by_list(self, size_list, num_case=1, inplace=True):
         else:
             return new_imgs
 
+    def resize_by_hw_range(self, rng, inplace=True):
```

Comment:
are resize_by_hw_range and resize_by_hw_list different to scale_by_range and scale_by_list?